.gitignore

```diff
@@ -44,5 +44,14 @@ runs/
 # default directory where logs go
 logs/
 
+# default directory where collections go
+collections/
+
+# default directory where indexes go
+indexes/
+
 # default output location of "Neural Hype" experiments: https://github.com/castorini/anserini/blob/master/docs/experiments-forum2018.md
-fine_tuning_results/
\ No newline at end of file
+fine_tuning_results/
+
+# directory where we keep throw-away Java classes that we don't want checked into the repo.
+src/main/java/io/anserini/scratch/
```

Comment:
Put a `.gitkeep` here too so that new users know the intent of the folder?

Reply:
👍